tests/functional/models/appflow-with-more-protocols/2020-08-23/endpoint-rule-set-1.json

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
Do we need this or the paginator file?

Reply:
Nope- vastly simplified the models for the test